workspace_host/interface.js

```diff
@@ -49,9 +49,9 @@ app.post('/', function(req, res) {
     const workspace_id = req.body.workspace_id;
     const action = req.body.action;
     const useInitialZip = _.get(req.body.options, 'useInitialZip', false);
-    if (workspace_id == undefined) {
+    if (typeof workspace_id === 'undefined') {
         res.status(500).send('Missing workspace_id');
-    } else if (action == undefined) {
+    } else if (typeof action === 'undefined') {
```

Comment:
A good JS pattern I often use in situations like this is `if (workspace_id == null)`. Using `== null` (two equal signs) will give `true` if `workspace_id` is either `undefined` or `null`, both of which it would be good to detect.